NEWS

```diff
@@ -1,6 +1,13 @@
 Unreleased
 ==========
 
+0.39.0
+======
+
+* [fix] fix the monkey patching with the asyncio hub
```

Comment:
Really it's only the first bullet item, I think.

Reply:
Second one is implementation detail, third one isn't true anymore because I came up with better unmonkeypatching.